api/src/main/java/org/openmrs/module/appointments/service/impl/TeleconsultationAppointmentService.java

```diff
@@ -1,14 +1,34 @@
 package org.openmrs.module.appointments.service.impl;
 
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.Patient;
+import org.openmrs.PatientIdentifier;
+import org.openmrs.api.PatientService;
 import org.openmrs.api.context.Context;
+import org.openmrs.module.appointments.model.AdhocTeleconsultationResponse;
 import org.openmrs.module.appointments.model.Appointment;
+import org.openmrs.module.appointments.notification.NotificationResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
 
 import java.text.MessageFormat;
+import java.util.List;
+import java.util.Random;
 
+@Component
 public class TeleconsultationAppointmentService {
 
     private final static String PROP_TC_SERVER = "bahmni.appointment.teleConsultation.serverUrlPattern";
+    private final static String ADHOC_TC_ID = "bahmni.adhoc.teleConsultation.id";
     private final static String DEFAULT_TC_SERVER_URL_PATTERN = "https://meet.jit.si/{0}";
+    private Log log = LogFactory.getLog(this.getClass());
+
+    @Autowired
```

Comment:
Please do not use autowiring for this service. use the XML /moduleApplicationContext.xml

Reply:
Changed

Reply:
@angshu : Please review and close. thanks!